Azure/communication-ui-library
# Bug 2564703 - [Call composite - control bar navigation]: Icon is not clearly visible on All buttons in aquatic mode under 'Calling window' page.

This PR is for Bug 2564703 - [Call composite - control bar navigation]: Icon is not clearly visible on All buttons in aquatic mode under 'Calling window' page.


## Issues Linked 
https://skype.visualstudio.com/SPOOL/_workitems/edit/2564703

## Changes
Added custom styling on buttons

## Checklist

Tested on Mac accessibility mode. 
Need testing on Windows

## Testing Steps
1. Open the above URL and login with v-credential.
2. 'Start or Join a call' page will be open.
3. Enter your name in 'Enter a name' edit field.
4. Navigate to 'Next Button' using tab key and activate it using 'Enter' key.
5. 'Start a call' page will be open.
6. Navigate to start call button using tab key and active it using enter key.
7. Call will be started and Calling window will be appear.
8. Navigate to 'Share' button using tab key.
9. Observe Keyboard focus boundary with respect to 'Share button in 'Aquatic' mode
